applications/stepswitch/src/stepswitch_util.erl

```diff
@@ -283,14 +281,15 @@ maybe_add_sip_headers(Endpoint, SIPHeaders) ->
 
 -spec maybe_endpoint_format_from(wh_json:object(), ne_binary(), api_binary()) ->
                                         wh_json:object().
-maybe_endpoint_format_from(Endpoint, Number, DefaultRealm) ->
+maybe_endpoint_format_from(Endpoint, Number, OffnetReq) ->
```

Comment:
the change of `DefaultRealm` to `OffnetReq` needs a
change in spec from `api_binary()` to `wapi_offnet_resource:req()`